### ISSUE TITLE
Spoofed Timezone x Request Date

### ISSUE THREAD

**Issue by Raprp** [dillbyrne/random-agent-spoofer#221]
Hi,

I configured my RAS to use another timezone, but at work today I logged at amazon S3 console and received an error saying that the difference between the current time and the request time is too big and the request will be ignored.

So my question is, when RAS changes the timezone offset, it also converts the request date from the local time to the fake timezone? For example, if I'm on Sydney and I set the timezone of Califonia, what will be the request date header value? Syney or Cali time?

I tryied to get the extension to look for myself, but I'm not very skilled in firefox extension development. 

Cheers,

**Comment by dillbyrne** [comment]
Hello @Raprp 

RAS does not change any date headers or anything like that. For now it only changes date and time strings. See the date handler portion of the [inject script](https://github.com/dillbyrne/random-agent-spoofer/blob/master/data/js/inject.js#L7) for exactly what it changes.

Initially I had it changing more than just the strings but it was causing page loading delays and timeouts.

**Comment by Raprp** [comment]
@dillbyrne,

Yeah, maybe messing with the request dates would only make navigation unstable.

I will take a look on the source code to see if a I have any ideas, but since I know nothing about extension development I doubt I will get anywhere =)

Thanks for the reply!

**Comment by Geyup** [comment]
I'm not sure about other systems, but in Unix/Linux as an example you can start Fireofx like this to spoof Time;

```TZ=Germany firefox```
